### ISSUE TITLE
Publish signed builds to GH Releases

### ISSUE THREAD

**Issue by abbyad** [medic/medic-collect#27]
As a follow up to https://github.com/medic/medic/issues/5479, we should use CI automation for putting signed builds in GitHub Releases.

Ideally we would handle #13 and #17 as part of this process, or have a newly defined for flavors.

**Comment by kennsippell** [comment]
This issue is waiting on the following information from @jillrshah. 

* The passwords for each flavor
* A list of the flavors which set `BIKRAM_SAMBAT: true`

**Comment by kennsippell** [comment]
@abbyad I just found all of [these](https://github.com/medic/medic-projects/tree/master/_medic-collect-flavors) configurations for medic-collect, which I didn't realise existed. Were you imaging we would build all of those flavors also when we change the medic-collect app - or can we address that in a new issue in a later release?

**Comment by abbyad** [comment]
I was thinking we'd only include these:
https://github.com/medic/medic-collect/blob/73bf9988c776902e368ea6ac4ad8f84dbbd146dd/build.gradle#L76-L114

Is that what you've been planning for too?

We should have a separate issue for cleaning up obsolete Collect flavor project files in medic-projects.

**Comment by newtewt** [comment]
@abbyad  Is there a way to tell in the app that collect was built for a partner? When clicking on Get Blank Form it seems to be directing to alpha which no longer exists in any of the apks. 

Also we are building for ancient versions of android based on the minsdk and targetsdk. I'll create new issues for these.

**Comment by kennsippell** [comment]
@abbyad Yep. As written we are building all flavors in the public repo.

**Comment by abbyad** [comment]
@newtewt you can see the build in the Collect app's general settings, or the Android app ID via the Android settings.

You can see the URLs via the [flavor's untranslated.xml file](https://github.com/medic/medic-projects/blob/ec6a8eb80f37b699ab6debd4379041bd94058359/_medic-collect-flavors/demo/res/values/untranslated.xml#L5). I don't see which one is alpha.dev though.

As for the older Android, that is expected as Collect has much lower minimum Android requirements. We can likely bump up the targetsdk while keeping the minsdk low.

**Comment by newtewt** [comment]
I'm able to launch and run the individual builds and tested a sample for to gamma.dev.
